hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
+  }
+
+  export interface VNode {
+    tag: string
+    data: VNodeData
+    children: string | VNode | Array<VNode>
+  }
+
+  export type viewFunction<State, Actions> = (
+    state: State,
+    actions: Actions
+  ) => VNode
+  export type emitFunction<Data> = (event: string, data?: Data) => Data
+  export type vnodeFunction = (
+    props: VNodeData,
+    children: string | VNode | Array<VNode>
```

Comment:
Is this a type for the [Stateless Components doc](https://github.com/hyperapp/hyperapp/blob/master/docs/stateless-components.md)? If I'm reading the code correctly, the last line in h.js (`tag(data, children)`) seems to say the children type would always be an array:
```typescript
children: Array<VNode | string>
```